### PR TITLE
Removed ocs-consumer, ocs-converged and ocs-converged-qe jobs

### PR DESCRIPTION
**Pull request by rohan47**
Removed ocs-consumer, ocs-converged and ocs-converged-qe jobs

Signed-off-by: Rohan Gupta <rohgupta@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml [hunk 1]
```diff
@@ -525,45 +525,6 @@ tests:
           cpu: 100m
           memory: 200Mi
     test:
-    - as: addon-ocs-consumer-tests
```
Comment (rohan47) [inline ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml:528]: @alfonsomthd  The ocs-consumer job was not removed after running `make jobs` after removing this job config.
Do I need to do it manually?

Comment (alfonsomthd) [reply ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml:528]: Then yes, delete it manually. You can also run `make update` (that also includes `make jobs`) to verify that all is correct or at least `make ci-operator-config` (also included in `make update`).

Comment (rohan47) [reply ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml:528]: Had to remove it manually, there were no changes after running `make update`.
Thanks

**Comment by alfonsomthd** [comment]
@MrSantamaria Can you please give a quick review?

**Comment by rohan47** [comment]
@MrSantamaria @mrbarge can you please give a quick review?

**Comment by alfonsomthd** [comment]
> @MrSantamaria Can you please give a quick review?

@jeremyeder con you help move this forward?

**Pull request merged**